articles/relnotes/index.md

```diff
@@ -10,10 +10,81 @@ uid: microsoft.quantum.relnotes
 
 # Microsoft Quantum Development Kit Release Notes
 
+# Version 0.6.1904
+
+*Release date: April 31, 2019 (May 1, 2019)*
+
+This release contains the following:
+- makes changes to the Q# language, 
+- restructures the Quantum Development Kit libraries, 
+- adds a newly contributed sample, and 
+- fixes bugs.  
+
+The changes are summarized here as well as instructions for upgrading your existing programs.  You can read more about these changes on devblogs.microsoft.com/qsharp.
+
+## Q# language syntax
+This release adds new Q# language syntax:
+* Add a [shorthand way to express variations of quantum operations](xref:microsoft.quantum.language.type-model#functors) (control and adjoints) with "+" operators.  The old syntax is deprecated.  Programs that use the old syntax (e.g., ": adjoint") will continue to work, but a compile time warning will be generated.  
+* Add a new operator for [copy-and-update](xref:microsoft.quantum.language.expressions#copy-and-update-expressions), "w/", can be used to express array creation as a modification of an existing array.
+* Add the common [apply-and-upate statement](xref:microsoft.quantum.qsharp-ref.statements#rebinding-of-mutable-symbols), e.g., "+=", "w/=".
+* Add a way to specify a short name for namespaces in [open directives](xref:microsoft.quantum.language.file-structure#open-directives).
+
+With this release, we no longer allow an array element to be specified on the left side of a set statement.  This is because that syntax implies that arrays are mutable when in fact, the result of the operation has always been the creation of a new array with the modification.  Instead, a compiler error will be generated with a suggestion to use the new copy-and-update operator, "w/", to accomplish the same result.  
```

Comment:
```suggestion
With this release, we no longer allow an array element to be specified on the left side of a set statement.  This is because that syntax implies that arrays are mutable when in fact, the result of the operation has always been the creation of a new array with the modification.  Instead, a compiler error will be generated with a suggestion to use the new copy-and-update operator, `w/`, to accomplish the same result.  
```